### ISSUE TITLE
Can someone tell  expected release date for eslint-sonarts and  depreciation date - tslint-sonarts ? 

### ISSUE THREAD

**Issue by Pooja123-gp** [SonarSource/SonarTS#900]
**I want to request a feature.**

**I want to report a bug.**

**SonarTS version:** <br>
**Node.js version:** <br>
**TypeScript version:** <br>

**SonarQube version:** <br>
**SonarLint version and used IDE:** <br>
**TSLint version:** <br>

**Rule key:** 

**Reproducer** 
```
// minimal reproducer if relevant
console.log("Hello, world"); // False-positive here
```

**Logs**
```
// logs of analysis if relevant
```

**Expected behavior**

**I want to provide the following feedback.**

**Rule key:** 

**Example**
